### ISSUE TITLE
set psp folder path before building

### ISSUE THREAD

**Issue by felafelfel** [hrydgard/ppsspp#15423]
### Compiler and build tool versions

cmake

### Operating system version

ubuntu latest

### Build commands used

im using ubuntu and github workflow 
every thing works great

### What happens

hello there
im trying to set psp folder path before  building app
how to disable dialog for seting psp folder path  and do this before building the app?

### PPSSPP version affected

latest

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] Make sure to run `git submodule update --init --recursive` before building.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try deleting the build directory and running the build again.
- [ ] Check GitHub Actions, which builds every merge and PR.
- [ ] Include logs and help us reproduce.

**Comment by unknownbrackets** [comment]
On Linux, you should not be asked to select a path.  This happens only when you're on Android.

On Android, it is required because the Android OS requires you to select and confirm permission grant for the path.  So it's not possible for PPSSPP to "default" to a path on storage.

The default path on Linux/POSIX is handled here:
https://github.com/hrydgard/ppsspp/blob/e88ddbb42a8a8a8699912eeed746b331c3eebdab/UI/NativeApp.cpp#L576-L584

Basically, it's `~/.config/ppsspp`.

-[Unknown]

**Comment by unknownbrackets** [comment]
As far as I know, you have to use the system dialog to select the path and allow it.  It's probably possible to make it pop up right away, not sure if the Android OS would reject a browse like that that started with no user interaction.  We don't do it because it's confusing and alarming to users.

-[Unknown]

**Comment by unknownbrackets** [comment]
You'll need to change the Android OS source code to modify that dialog.

-[Unknown]
